backend/app/controllers/application_controller.rb

```diff
@@ -1,2 +1,9 @@
 class ApplicationController < ActionController::API
+  before_action :random_errors
+
+  def random_errors
+    if rand(1..10) == 1
+      render json: { error: 'Unknown Error Occurred (did you handle it??)' }, status: :internal_server_error
+    end
+  end
```

Comment:
Is this too hard/confusing?

Basically every endpoint will randomly return a 500 error, roughly 10% of the time.

I wanted to add some proof that they're doing error handling on the frontend, but I could also cut this if we think that's too tough.

Reply:
I prefer using a parameter where we could do this. Lets say: 
If invoice id = -1 then throw an error